content/2021/pages/4.md

```diff
@@ -0,0 +1,113 @@
+---
+section: Politechnika w liczbach
+bgcImage: /images
+pageNumber: 4
+---
+
+<div>
+<h2>Największa <br> uczelnia technologiczna <br> w regionie</h2><br>
+Politechnika Białostocka powstała w 1949 r.<br>
+Mamy dwa kampusy w centrum Białegostoku<br>
+i pobliskim Kleosinie, nowoczesne pracownie<br>
+i unikalną w skali kraju aparaturę badawczą.<br>
+Biblioteka PB ma największy w północno-<br>
+-wschodniej Polsce księgozbiór naukowo-<br>
+-techniczny, który liczy blisko 400 tys. pozycji,<br>
+i doskonałe warunki do nauki.<br>
+<i class="fab fa-youtube"></i>Obejrzyj wideo:<br>
+<a hrfer="https://bit.ly/politechnika-bialostocka">bit.ly/politechnika-bialostocka</a><br><br><br>
+</div>
+
+<div>  
+<h2>29 kierunków <br> studiów</h2>
+Programy naszych<br>
+kierunków były konsultowane<br>
+z największymi<br>
+pracodawcami regionu.<br><br><br>
+</div>
+
+<div>
+<h2>7 tysięcy <br> studentów</h2>
+Blisko 60 aktywnych<br>
+kół naukowych<br>
+i organizacji<br>
+studenckich.<br><br><br>
+</div>
+
+<div>
+<h2>Wsparcie <br> materialne</h2>
+Studenci Politechniki Białostockiej<br>
+mogą starać się<br>
+o stypendium rektora, stypendia<br>
+socjalne, stypendium<br>
+mieszkaniowe „Akademiki za<br>
+dobre wyniki”, stypendium<br>
+Programu „Odkrywcy<br>
+Diamentów”.<br><br><br> 
+</div>
+
+<div>
+<h2>Studia bez granic</h2>
+Umiędzynarodowienie studiów i rozwijanie współpracy z uczelniami<br>
+partnerskimi na całym świecie to zadanie priorytetowe Politechniki<br>
+Białostockiej. Prowadzimy intensywną współpracę z Chinami, a dzięki<br>
+ponad 300 aktywnym porozumieniom w programie wymiany studenckiej<br>
+Erasmus+ nasi studenci mogą podjąć naukę w uczelniach na całym<br>
+świecie. Aktualnie mamy 23 umowy o podwójnym dyplomowaniu<br>
+z uczelniami w Chinach, Danii, Francji, Hiszpanii, Portugalii, Ukrainie,<br>
+Kazachstanie, Kirgistanie i Rosji. Organizujemy międzynarodowe szkoły<br>
+letnie z zakresu logistyki, architektury krajobrazu, budownictwa<br>
+i inżynierii środowiska.<br>
+<i class="fab fa-youtube"></i>Obejrzyj wideo:<br>
+<a href="https://bit.ly/studia_bez_granic">https://bit.ly/studia_bez_granic</a><br><br><br>
+
+</div>
+
+<div>
+<h2>630 nauczycieli
+akademickich</h2>
+W gronie naszych wykładowców
+są specjaliści w swoich branżach,
+eksperci współpracujący z biznesem
+oraz instytucjami publicznymi,
+stowarzyszeniami branżowymi, m.in.
+Stowarzyszeniem Elektryków Polskich,
+Polskim Towarzystwem Elektrotechniki
+Teoretycznej i Stosowanej, Polskim
+Związkiem Inżynierów i Techników Budownictwa,
+Stowarzyszeniem Architektów
+Polskich, Polskim Towarzystwem Matematycznym.
+Nauczyciele akademiccy PB pracują
+w zespołach doradczych Ministra Nauki
+i Szkolnictwa Wyższego, Radzie Narodowego
+Centrum Badań i Rozwoju, są członkami
```

Comment:
ani divów ani li. To miałbyć plain markdown, a nie skopiowane z informatora styl. Nasz informator będzie wyglądał inaczej